### PR TITLE
fix: popup

### PR DESCRIPTION
**Pull request by agrim-deriv**

### CODE REVIEW

**Review by Nuzhy-Deriv — APPROVED** [review]

File: src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx [hunk 1]
```diff
@@ -39,19 +39,25 @@ export const AppRegisterSuccessModal = ({
...
+              <Text>Creating an API token to use with your application.</Text>
+            </li>
+            <li>
+              <Text>Adding OAuth authentication in your app.</Text>
+            </li>
+            <li>
+              <Text>Selecting the scopes of OAuth authorisation for your app.</Text>
+            </li>
+          </ul>
+          <div>
```
Comment (ahmed-deriv) [inline src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx:58]: do we require this extra `div` element to wrap. ? `Text` isn't a block element ?

File: src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx [hunk 2]
```diff
@@ -39,19 +39,25 @@ export const AppRegisterSuccessModal = ({
...
-            <br></br>Optimise your app&apos;s capabilities by:
-            <ul>
-              <li>Creating an API token to use with your application.</li>
-              <li>Adding OAuth authentication in your app.</li>
-              <li>Selecting the scopes of OAuth authorisation for your app.</li>
-            </ul>
-            <div>Note: You can make these changes later through the dashboard.</div>
-          </span>
+      <div className='modal__content'>
+        <Heading.H4>Application registered successfully!</Heading.H4>
```
Comment (ahmed-deriv) [inline src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx:43]: @agrim-deriv  We don't need translation here in this component ?

File: src/styles/index.scss [hunk 3]
```diff
@@ -242,13 +242,6 @@ div[class*='sidebarViewport'] {
 
   h4 {
     margin-bottom: 32px;
-    color: var(--core-color-solid-slate-1400);
```
Comment (ahmed-deriv) [inline src/styles/index.scss:245]: any particular reason to remove style from global file ? is there any default `SCSS` applied from quill ?

**Review by ahmed-deriv — CHANGES_REQUESTED** [review]

File: src/styles/index.scss (hunk 3 above)
Comment (Nuzhy-Deriv) [reply src/styles/index.scss:245]: yes, quill got the default color

Comment (ahmed-deriv) [reply src/styles/index.scss:245]: what about font-size @agrim-deriv ? are we using default or have configure somewhere else ?

Comment (Nuzhy-Deriv) [reply src/styles/index.scss:245]: we have default font-sizes as well in quill.

Comment (ahmed-deriv) [reply src/styles/index.scss:245]: In this case we are not using quill heading and text. As its our old code and we haven't updated there so I we remove this styling would that work fine as a before or it has a chance that these tags would have different behavior than quill `H4` and `Text` ?  
image

File: src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx (hunk 1 above)
Comment (agrim-deriv) [reply src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx]: done

File: src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx (hunk 2 above)
Comment (agrim-deriv) [reply src/features/dashboard/components/Modals/AppRegisterSuccessModal/index.tsx]: done

File: src/styles/index.scss (hunk 3 above)
Comment (agrim-deriv) [reply src/styles/index.scss:245]: Nope the behavior is the same. Refer to https://quill-ui.pages.dev/?path=/docs/components-typography-heading--playground and to our figma https://www.figma.com/design/vxKXqGpgiwUpHnghfL4FjJ/API---App-Registration?node-id=10317-32846&node-type=instance&m=dev

**Review by prince-deriv — APPROVED** [review]

**Review by ahmed-deriv — APPROVED** [review]

**Pull request merged**